packages/picasso-codemod/src/v17.0.0/typography-sizes/typography-sizes.ts

```diff
@@ -10,8 +13,14 @@ const getNewSize = (oldSize: string): string => {
       return oldSize
   }
 }
-
-let affectedComponents = ['Typography', 'TypographyOverflow', 'Amount']
+// exclude components with `variant` prop other than body
+const shouldBeExcluded = (attributes: JSXAttribute[]) =>
+  attributes.find(
```

Comment:
`find` returns an item matching the condition. `some` will return you `true` or `false`.

```suggestion
  attributes.some(
```

Reply:
good idea ✅  thanks